web/src/features/panels/zone/ZoneDetails.tsx

```diff
@@ -105,29 +149,7 @@ export default function ZoneDetails(): JSX.Element {
           zoneDataStatus !== ZoneDataStatus.AGGREGATE_DISABLED && (
             <DisplayByEmissionToggle />
           )}
-        <ZoneDetailsContent
-          isLoading={isLoading}
-          isError={isError}
-          zoneDataStatus={zoneDataStatus}
-        >
-          <BarBreakdownChart hasEstimationPill={hasEstimationPill} />
-          <ApiButton backgroundClasses="mt-3 mb-1" type="primary" />
-          {zoneDataStatus === ZoneDataStatus.AVAILABLE && (
-            <AreaGraphContainer
-              datetimes={datetimes}
-              timeRange={timeRange}
-              displayByEmissions={displayByEmissions}
-            />
-          )}
-
-          <MethodologyCard />
-          <HorizontalDivider />
-          <div className="flex items-center justify-between gap-2">
-            <div className="text-sm font-semibold">{t('country-panel.forecastCta')}</div>
-            <ApiButton size="sm" />
-          </div>
-          <Attribution zoneId={zoneId} />
-        </ZoneDetailsContent>
+        {zoneDetailsContent}
```

Comment:
Moving this component into a `useMemo` requires moving it before any `return` statement, which required me to move some declarations further up the file.